src/rules/rule.ts

```diff
@@ -0,0 +1,42 @@
+const BLOCK_RULE = 'block';
```

Comment:
Don't use global variables. Encapsulate it inside your class as `public static readonly` variables.